src/components/skip/skip.js

```diff
@@ -43,6 +43,7 @@ class Skip extends Component {
    */
   onKeyDown = (e: KeyboardEvent): void => {
     if (e.keyCode === KeyMap.ENTER || e.keyCode === KeyMap.SPACE) {
+      e.preventDefault();
       this.props.onClick();
```

Comment:
1. is the order of the e.preventDefault(); before on purpose?
2. Dont we need also to stop propagate events?